ftw/news/browser/resources/news.scss

```diff
@@ -21,18 +24,30 @@ $leadimage-width: $scale-preview-width;
       @include screen-small() {
         float: left;
         > img {
-          max-width: $leadimage-width;
+          max-width: $leadimage-max-width;
         }
       }
+
+      background: url("++resource++ftw.news.resources/newspaper.svg") center / 50% no-repeat;
```

Comment:
Please use the `$portal-url` variable as basis for the url on order to make it cacheable.